### ISSUE TITLE
Cant delete adb

### ISSUE THREAD

**Issue by han73r** [fpv-wtf/wtfos-configurator#399]
### Device

Goggles V1

### Operating System (Including version)

Windows 10 Pro 22H2 19045.3208

### Browser

Google Chrome

### Trace ID (bottom left of the configurator screen)

f392646a-7a8d-4574-bb83-238c57b33f9a

### Describe the issue

Hi!
I use my caddx vista with rooted DJI goggles V1. No I need to pair caddx vista with DJI integra. So, I have to update firemware and I try to remove wtfos and adb from my caddx vista but configurator wrote that I haven't any adb.

Also I cand reinstall WTFOS because buttons are inclickable. I added example below.

I tried it on a dfferent machines, browsers: Chrome and Edge. With the same result unfortantly :(

Tried to check what is installed:
root@pigeon_wm150_tiny:/ # opkg list-installed
busybox - 1.35.0-3
dinit - 0.14.0-r2
entware-opt - 227000-3
entware-release - 1.0-2
entware-upgrade - 1.0-1
findutils - 4.9.0-1
grep - 3.8-2
jq - 1.6-2
jsonfilter - 2018-02-04-c7e938d6-1
libc - 2.27-11
libgcc - 8.4.0-11
libjson-c - 0.16-2
libpcre2 - 10.37-2
libpthread - 2.27-11
librt - 2.27-11
libssp - 8.4.0-11
libstdcpp - 8.4.0-11
libubox - 2022-09-27-ea560134-1
locales - 2.27-9
opkg - 2022-02-24-d038e5b6-1
terminfo - 6.3-1a
wtfos - 0.3.4
wtfos-opkg-config - 1.1.1
wtfos-package-config - 0.1.1
wtfos-system - 0.2.5
zoneinfo-asia - 2022f-1
zoneinfo-europe - 2022f-1
root@pigeon_wm150_tiny:/ #

WTFOS PAGE
![WTFOS](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/50344edd-1b3a-4f65-9796-5c67173e1aa1)

CLI
![CLI](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/c0075707-d4c9-4353-87ab-169a684ca589)

HOME
![Connection](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/a2168edb-e955-49de-b76d-675ca9f028bd)

DJI
![DJI](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/1a2027bd-97ab-4cbf-8ca9-8b275bb35b6f)

Packages
![Packages](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/971703a5-798a-4437-9eb6-efaf41448406)

Root
![Root](https://github.com/fpv-wtf/wtfos-configurator/assets/92275256/bcadbea5-7005-4b22-8f13-6711e1123947)

**Comment by stylesuxx** [comment]
Startup and removal tiles are grayed out - this is an indicator that the dinit binary is missing. Seems you are in a bit of a broken state. You could try to remove wtfos via opkg, cleanly re-install and then use the regular removal process to get a clean uninstall.

**Comment by han73r** [comment]
I think same. How can I remove wtfos via opkg?

**Comment by han73r** [comment]
Id didn't work too(

root@pigeon_wm150_tiny:/ # opkg remove wtfos
No packages removed.
Collected errors:
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/dinit.list: No such file or directory.
root@pigeon_wm150_tiny:/ # 

or

root@pigeon_wm150_tiny:/ # opkg install wtfos
Package wtfos version 0.3.4 has no valid architecture, ignoring.
Package wtfos version 0.3.4 has no valid architecture, ignoring.
Installing wtfos (0.3.4) to root...
Downloading http://repo.fpv.wtf/pigeon/wtfos_0.3.4_pigeon-airside.ipk
Configuring wtfos.
Collected errors:
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/dinit.list: No such file or directory.
root@pigeon_wm150_tiny:/ #

**Comment by han73r** [comment]
1|root@pigeon_wm150_tiny:/ # cat /blackbox/dinit.log    
Hundred rows like that
> dinit: service boot started.
dinit: service boot started.
dinit: service msp-osd-airside started.
dinit: service wtfos-auto-naco started.

And Then

> dinit: service msp-osd-airside stopped.
dinit: Service wtfos-auto-naco stop command failed with exit code 127
dinit: service wtfos-auto-naco stopped.
dinit: service boot stopped.

root@pigeon_wm150_tiny:/ # dinitctl list

> /system/bin/sh: dinitctl: not found

127|root@pigeon_wm150_tiny:/ # df /blackbox/

> Filesystem               Size     Used     Free   Blksize
> /blackbox/               1.5G   540.7M   971.2M   4096

root@pigeon_wm150_tiny:/ # df /

> Filesystem               Size     Used     Free   Blksize
> /                      308.8M     1.4M   307.4M   4096

root@pigeon_wm150_tiny:/ # cat /blackbox/wtfos.log
Hunred rows like that and 

> wtfos: system loopmounted
> wtfos: ran dji startup script
> wtfos: started entware unslung

**Comment by han73r** [comment]
Yes! It's worked! 
I used wtfos-remove-adb next. 
And I've already installed v01.01.0000 via DJI Assistant. 
Thank you so much!

**Comment by stylesuxx** [comment]
Very welcome, I'll mark this as resolved.

**Comment by holdmebeer** [comment]
Hello I am having a hard time getting the new firmware’s to show up in the dji assistant 2 app. I tried to remove errors after a failed firmware update that got stuck at 87% several times once I removed wtfos I was able to install 1.0.7.0000 but I’m sure this isn’t the correct newest formate. I was able to connect to my 03 air units and vista air unit but I’m trying to connect to my 04 air unit now and I forgot the process and feel like I may have made a mess of things. Currently reading this I have reinstalled wtfos but when I try to remove add both before reinstalling wtfos and after it says thsy it can’t find any adb.. and the current firmware is showing as an upgrade to 1.07.0000  with refresh on 1.06.0000 and I am using the dji fpv goggles v2

![Image](https://github.com/user-attachments/assets/87c5d8cc-2dfc-4380-8294-0631fadc3922)

![Image](https://github.com/user-attachments/assets/49069976-f908-407c-a048-33ebcad376de)

![Image](https://github.com/user-attachments/assets/a65e2b10-fa1a-4daf-9454-7c586778c298)
